docs/loading/Spark-connector-starrocks.md

```diff
@@ -2,12 +2,22 @@
 
 StarRocks provides a self-developed connector named StarRocks Connector for Apache Spark™ (Spark connector for short) to help you load data into a StarRocks table by using Spark. The basic principle is to accumulate the data and then load it all at a time into StarRocks through [STREAM LOAD](../sql-reference/sql-statements/data-manipulation/STREAM%20LOAD.md). The Spark connector is implemented based on Spark DataSource V2. A DataSource can be created by using Spark DataFrames or Spark SQL. And both batch and structured streaming modes are supported.
 
+> **NOTICE**
+>
+> Only users with the SELECT and INSERT privileges on a StarRocks table can load data into this table. You can follow the instructions provided in [GRANT](../sql-reference/sql-statements/account-management/GRANT.md) to grant the required privileges.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/624619da-200d-4fc6-bfad-ae0decd21c02)
-> grant these privileges to a user